x-pack/plugins/apm/public/components/app/Home/index.tsx

```diff
@@ -106,6 +116,15 @@ export function Home({ tab }: Props) {
               </EuiButtonEmpty>
             </SettingsLink>
           </EuiFlexItem>
+          {isAlertingAvailable && (
+            <EuiFlexItem grow={false}>
+              <AlertIntegrations
```

Comment:
I know you didn't come up with the name but "AlertIntegrations" is super generic. Can we call it something more accurate:

```suggestion
              <AlertingPopover
```

or 

```suggestion
              <AlertingFlyout
```

or 

```suggestion
              <AlertingPopoverAndFlyout
```

Reply:
`AlertingPopoverAndFlyout` it is!!!